### DISCUSSION TITLE
articles/paternity

### DISCUSSION THREAD

**Discussion by giscus** [hutusi/hutusi.github.com#166] [Comments]
# articles/paternity

妻子的预产期是5月24日，但临近预产期肚子却毫无要生产的迹象，宝宝在肚子里每天依然活动的很频繁，而且妻子也没有了肚子痛的感觉，散步的时候甚至健步如飞。终于在5月25日产检时，在妻子的在三请求下，医生答应剖腹产，时间5月27日。

https://hutusi.com/articles/paternity

**Comment by Aaron-Carol** [comment]
写的真好呀

**Reply by hutusi** [reply]
谢谢，回头看看自己以前的文章，也还挺有意思😁
